crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/service/IncomeEvidenceValidationService.java

```diff
@@ -59,13 +59,13 @@ public void checkEvidenceDueDates(LocalDate evidenceDueDate, LocalDate existingE
 
     public void checkExtraEvidenceDescriptions(List<ApiIncomeEvidence> incomeEvidences) {
         incomeEvidences.stream()
-            .filter(apiIncomeEvidence -> EXTRA_EVIDENCES.contains(apiIncomeEvidence.getEvidenceType()))
-            .forEach(apiIncomeEvidence -> {
-                    if (StringUtils.isBlank(apiIncomeEvidence.getDescription())) {
-                        throw new CrimeEvidenceDataException(MISSING_OTHER_EVIDENCE_DESCRIPTION);
-                    }
-                }
-            );
+                .filter(apiIncomeEvidence -> EXTRA_EVIDENCES.contains(apiIncomeEvidence.getEvidenceType()))
+                .forEach(apiIncomeEvidence -> {
+                            if (StringUtils.isBlank(apiIncomeEvidence.getDescription())) {
+                                throw new CrimeEvidenceDataException(MISSING_OTHER_EVIDENCE_DESCRIPTION);
+                            }
+                        }
+                );
```

Comment:
Would you be able to revert this change please?

Reply:
Here No changes. I just formatted code

Reply:
Sorry, that was what I meant. If there are no changes to the code then I don't think we should be reformatting it, unless we're doing a separate piece of clean-up work. I'd also say that the extra spacing doesn't really seem right, I think maybe this is one as a team we need to agree on for it to work.

Reply:
As discussed also, I think the formatting issue is a team discussion and requires everyone to be on board and using the same standards if we are to achieve consistency. Will let this one in in this PR but ideally if we can avoid updating formatting on code files we're not changing that would be the preference 🙂